sotodlib/coords/det_match.py

```diff
@@ -0,0 +1,670 @@
+import warnings
+from dataclasses import dataclass, fields, asdict
+from typing import List, Optional, Tuple, Iterator
+from copy import deepcopy
+import h5py
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.optimize import linear_sum_assignment
+
+from sotodlib.coords import optics
+from sotodlib.core import metadata
+from sotodlib.io.metadata import write_dataset
+
+warnings.filterwarnings(action='ignore',
+                        message='overflow encountered in exp*', category=RuntimeWarning)
```

Comment:
Can you use a more targeted context manager instead of just turning these off for everything?